js/ui/camera.js

```diff
@@ -16,7 +16,8 @@ var Point = require('point-geometry');
  * @property {Array} center Longitude and latitude (passed as `[lng, lat]`)
  * @property {number} zoom Map zoom level
  * @property {number} bearing Map rotation bearing in degrees counter-clockwise from north
- * @property {number} pitch The angle at which the camera is looking at the ground
+ * @property {number} pitch The angle in degrees at which the camera is looking at the ground
+ * @property {Array} around If zooming, the zoom center (defaults to map center, passed as `[lng, lat]`)
```

Comment:
> It's probably a good idea to add a @typedef {Array<number>|LngLat} LngLatish definition where we can document this precisely and talk about coordinate order in gl-js (#739), and use that everywhere that a LngLat or array may be passed.

[Leaflet's documentation](http://leafletjs.com/reference.html#latlng) does a really nice job explaining that everywhere that accepts `LatLng` can also accept array or object forms.  

I think I could just use similar language in `geo/lng_lat.js` and just drop all the unnecessary  `Array<number>` type docs.